### DISCUSSION TITLE
MCCHECK NOT READY

### DISCUSSION THREAD

**Discussion by mflynn-lanl** [microbiomedata/nmdc-edge#236] [General]
I've tried creating a couple of Magic Castle clusters, one with one node, another with three nodes. However, even after more than 24 hours `mccheck` returns `NOT READY`

**Comment by edwins** [comment]
@mflynn-lanl I'll take a look

**Reply by edwins** [reply]
@mflynn-lanl I apologize for the delay. Can you try to launch a new cluster now? There was a configuration value that I should have pinned -- because the configuration value was unpinned, mc was pulling from the development branch.

**Comment by mflynn-lanl** [comment]
mccheck returns `READY` but the guest user doesn't exist:
```
[mcflynn@login1 ~]$ sudo su - user
su: user user does not exist
```
however, the guest user is in the `accounts.txt` file:
```
[mcflynn@login1 ~]$ cat accounts.txt

The following is the account information:

usernames: user[1-1]
password: rarely.indirectly.composed.mayfly
```

**Reply by edwins** [reply]
@mflynn-lanl you can try again.

**Comment by mflynn-lanl** [comment]
The guest user is still not available:
```
[mcflynn@login1 ~]$ sudo su - user
su: user user does not exist
[mcflynn@login1 ~]$ cat accounts.txt

The following is the account information:

usernames: user[1-1]
```

**Reply by edwins** [reply]
It looks like the guest users are created in Alma9 but not Alma8 (at least in my testing). Are you able to switch to Alma9?

**Reply by mflynn-lanl** [reply]
Weird, that fixed it but it didn't make a difference before. The cluster we've been using is Alma8 and `user` existed. I created a new one with Alma9 and I can `sudo su - user`

**Reply by edwins** [reply]
I'm not sure what's changed, but if you can use Alma9, I will investigate when I get the chance.

**Reply by edwins** [reply]
@mflynn-lanl This was just posted on magic castle slack and issues board about Alma8. It looks like FreeIPA is the culprit.

https://github.com/ComputeCanada/magic_castle/issues/315
